fastlane/lib/assets/DefaultFastfileTemplate

```diff
@@ -11,6 +11,6 @@
 
 default_platform(:ios)
 
-platform :ios do
+platform(:ios) do
```

Comment:
I do have to say, that we should make an exception for `platform` and `lane`, as it looks much nicer without brackets, and we don't need to let people know it's actually a method call

Reply:
Sure